shm/src/player.cpp

```diff
@@ -14,10 +14,11 @@ void Player::addMoney(const size_t& money) {
 }
 
 size_t Player::countSpace() {
-    size_t totalAmount = std::accumulate(ship_->getCargo().begin(), ship_->getCargo().end(), 0,
+    /*size_t totalAmount = std::accumulate(ship_->getCargo().begin(), ship_->getCargo().end(), 0,
                                          [](const Cargo& item1, const Cargo& item2) {
                                              return item1.getAmount() + item2.getAmount();
                                          });
 
-    return ship_->getCapacity() - totalAmount;
+    return ship_->getCapacity() - totalAmount;*/
+    return 10;
```

Comment:
Easy solution:
```
size_t totalAmount = 0;
for (auto it = ship_->getCargo().begin(); it != ship_->getCargo().end(); ++it) {
    totalAmount += (*it)->getAmount();
}
return ship_->getCapacity() - totalAmount;
```

Reply:
Player is work of Jakub, it's here by mistake, DONT MERGE PLAYER FROM THIS BRANCH